<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_win</class>
 <widget class="QMainWindow" name="main_win">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>601</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>画板</string>
  </property>
  <property name="windowIcon">
   <iconset resource="img_rcc.qrc">
    <normaloff>:/Icon/board.png</normaloff>:/Icon/board.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="line_action"/>
   <addaction name="round_action"/>
   <addaction name="fill_action"/>
   <addaction name="separator"/>
   <addaction name="load_action"/>
   <addaction name="ehm_action"/>
   <addaction name="canny_action"/>
   <addaction name="separator"/>
   <addaction name="clear_action"/>
  </widget>
  <action name="line_action">
   <property name="icon">
    <iconset resource="img_rcc.qrc">
     <normaloff>:/Icon/Line.png</normaloff>:/Icon/Line.png</iconset>
   </property>
   <property name="text">
    <string>线</string>
   </property>
   <property name="toolTip">
    <string>线</string>
   </property>
  </action>
  <action name="round_action">
   <property name="icon">
    <iconset resource="img_rcc.qrc">
     <normaloff>:/Icon/Circle.png</normaloff>:/Icon/Circle.png</iconset>
   </property>
   <property name="text">
    <string>圆</string>
   </property>
   <property name="toolTip">
    <string>圆</string>
   </property>
  </action>
  <action name="Hexagon_action">
   <property name="icon">
    <iconset resource="img_rcc.qrc">
     <normaloff>:/Icon/Hexagon.png</normaloff>:/Icon/Hexagon.png</iconset>
   </property>
   <property name="text">
    <string>六边形</string>
   </property>
  </action>
  <action name="open_action">
   <property name="text">
    <string>打开</string>
   </property>
   <property name="toolTip">
    <string>打开</string>
   </property>
  </action>
  <action name="save_action">
   <property name="text">
    <string>保存</string>
   </property>
   <property name="toolTip">
    <string>保存</string>
   </property>
  </action>
  <action name="clear_action">
   <property name="icon">
    <iconset resource="img_rcc.qrc">
     <normaloff>:/Icon/clear.png</normaloff>:/Icon/clear.png</iconset>
   </property>
   <property name="text">
    <string>清空</string>
   </property>
  </action>
  <action name="fill_action">
   <property name="icon">
    <iconset resource="img_rcc.qrc">
     <normaloff>:/Icon/Fill.png</normaloff>:/Icon/Fill.png</iconset>
   </property>
   <property name="text">
    <string>填充</string>
   </property>
  </action>
  <action name="load_action">
   <property name="icon">
    <iconset resource="img_rcc.qrc">
     <normaloff>:/Icon/Add.png</normaloff>:/Icon/Add.png</iconset>
   </property>
   <property name="text">
    <string>载入图像</string>
   </property>
  </action>
  <action name="ehm_action">
   <property name="icon">
    <iconset resource="img_rcc.qrc">
     <normaloff>:/Icon/Edit.png</normaloff>:/Icon/Edit.png</iconset>
   </property>
   <property name="text">
    <string>图像增强</string>
   </property>
  </action>
  <action name="canny_action">
   <property name="icon">
    <iconset resource="img_rcc.qrc">
     <normaloff>:/Icon/Edit.png</normaloff>:/Icon/Edit.png</iconset>
   </property>
   <property name="text">
    <string>边缘检测</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="img_rcc.qrc"/>
 </resources>
 <connections/>
</ui>
